consultation_analyser/pipeline/processing.py

```diff
@@ -60,7 +60,7 @@ def run_processing_pipeline(consultation):
         command = {
             "command": [
                 "venv/bin/django-admin",
-                "run_ml_pipeline",
+                "generate_themes",
```

Comment:
I think this might error if you run more than once?

Locally, I have run `poetry run python manage.py generate_themes --consultation_slug="consultation-slug-2024-06-27-13-21-14"` (for an existing consultation).

And I get this error: 
```
django.db.utils.IntegrityError: duplicate key value violates unique constraint "unique_up_to_question"
DETAIL:  Key (topic_id, question_id)=(0, f944f253-7130-42be-8de0-3dc8339d4516) already exists.
```

Reply:
This is on saving themes. Not quite sure why and potentially will change when we move to new models.

Reply:
Fine running generate themes twice in dev so that's a bit mysterious.

Reply:
great catch. investigating